test/e2e/specs/fakerp/admin.go

```diff
@@ -103,4 +134,182 @@ var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader][Fake]"
 		By("comparing the update blob before and after an update")
 		Expect(reflect.DeepEqual(before, after)).To(Equal(true))
 	})
+
+	It("should be able to configure azure file persistent volumes", func() {
+		const (
+			prefix = "azure-file"
+		)
+
+		var (
+			pvName string
+		)
+
+		pvName = fmt.Sprintf("%s-pv", prefix)
+		// need a way to lessen the chances of name collision in storage account names
+		// and also they must lowercase letters and numbers
+		clusterSHA := sha1.Sum([]byte(azCli.Config.ResourceGroup))
+		// storage account names are capped at 24 chars
+		accountName := fmt.Sprintf("azf%x", clusterSHA)[:24]
+
+		// create storage accounts and get their keys
+		By(fmt.Sprintf("Creating %s storage account", accountName))
```

Comment:
should be in the BeforeEach, with a matching delete in the AfterEach

Reply:
Somewhat disagree on this: creating a storage account for all other specs in the test suite even if they don't need it will just add up to the overall test time (creating the SA is slow)
I have added a new context to wrap the azure storage account tests in, and have put the test in there with the appropriate before/after each blocks